docs/careers.md

```diff
@@ -0,0 +1,44 @@
+# careers
+
+## Career Development Services (CDS)
+Career Development Services (CDS) office at IIT Gandhinagar is committed to empower the students to discover their true potential and help them to pursue their dreams in all possible immediate career verticals including Entrepreneurship, Placement, Higher Study and Competitive Examinations. Students can avail CDS resources that help them to explore and plan their career effectively. Office also conducts various sessions that help students in career counselling, resume preparation, screening tests, group discussion, interview process, personality development among others. CDS provides a platform where opportunity taker and opportunity giver can come together to explore avenues of a fruitful and synergistic alliance. We are proud of several such associations in the past in the form of internships, industry visits and placements for which we are grateful to participating organizations both from India and abroad. 
+
+## Internship
+##### Demography - B.Tech 2026
+![Internship Image](https://github.com/yuvraj-rathod-1202/metaiitgn/blob/academic-dean's-list/docs/career%20images/internship%2826%29.png?raw=true)
+
+##### Timeline
+![Internship Timeline](https://github.com/yuvraj-rathod-1202/metaiitgn/blob/academic-dean's-list/docs/career%20images/timeline%2826%29.png?raw=true)
+
+##### Academic Internship
+![academic internship](https://github.com/yuvraj-rathod-1202/metaiitgn/blob/academic-dean's-list/docs/career%20images/univercity%20intern.png?raw=true)
```

Comment:
Since you have added images in another folder, you could link them with relative paths rather than providing a github image link. Also in general try to have the image name without spaces for easy path linking.

```suggestion
## Internship

### Demography - B.Tech 2026
![Internship Image](./career%20images/internship%2826%29.png)

### Timeline
![Internship Timeline](./career%20images/timeline%2826%29.png)

### Academic Internship
![academic internship](./career%20images/univercity%20intern.png)
```